_DeploymentAndDistroPackaging/ansible/README.md

```diff
@@ -70,13 +59,27 @@ before proceeding to the next step.
 
 ---
 
+## Start the deployment container with your setup
+```
+$ docker run --privileged -v /path/to/your/.ssh/key:/root/.ssh/id_rsa \
+             -v $(pwd):/root/ciao \
+             -it clearlinux/ciao-deploy
+```
+
 ### Run the playbook
 
     ansible-playbook -i hosts ciao.yml
```

Comment:
This won't work as there are some symlinks in `./ciao/_DeploymentAndDistroPackaging/ansible` asking for files out of the `./ciao/_DeploymentAndDistroPackaging` folder...

Updated the PR to document that the whole `./ciao` repo should be mapped into the container instead.